core/frontend/README.md

```diff
@@ -0,0 +1,3 @@
+# CompanionOS
```

Comment:
Companion OS is what we were considering calling the project as a whole, not the frontend. I think the frontend could be just "frontend". And do we even need a readme here?

Reply:
Got it wrong on the meeting.
Let's stay with frontend.